### PR TITLE
add Mesh documentation in Get Started

### PR DESCRIPTION
**Pull request by jinglescode**
# Add documentation

Under `/docs/get-started/mesh`, added 8 pages of docs, this valuable information is helpful for developers to build apps on Cardano.

# Add Mesh as favorite

Mesh has been used and loved by students at Gimbalabs, new developers and experienced developers. I would like to suggest this to be tagged under `favorite`.

Asking for support for this PR:

@rcmorano @mmahut

### CODE REVIEW

File: docs/get-started/mesh/get-started.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+id: get-started
+sidebar_position: 2
+title: Get Starter - Mesh SDK (Open-Source Library for Building Web3 Apps on the Cardano Blockchain)
```
Comment (rdlrt) [inline docs/get-started/mesh/get-started.md:4]: Typo

File: src/data/builder-tools.js [hunk 2]
```diff
@@ -774,9 +774,9 @@ export const Showcases = [
     title: "Mesh",
     description: "An Open-Source library by Martify Labs with the purpose of making building dApps on Cardano easier than ever. The Mesh library is engineered to simplify the development and implementation of blockchain technology, so you can focus on your project.",
     preview: require("./builder-tools/mesh.png"),
-    website: "https://mesh.martify.io/",
+    website: "https://meshjs.dev/",
     getstarted: null,
```
Comment (rdlrt) [inline src/data/builder-tools.js:778]: You'd want to update getstarted reference here

**Review by rdlrt — COMMENTED** [review]
Would be welcome addition to `Favourites` IMO, noticed couple of minor typos while [quick] browsing

**Review by rphair — APPROVED** [review]
wow, this is a nice sheaf of documentation 🤓 ... I've double checked in a local build because of the extensive changes to the nav & everything looks good ✔️ and I agree with the rationale about this being marked as a `favorite`.  Nice touch adding the Open Graph image 😇

**Comment by rphair** [comment]
p.s. it appears the `getstarted:` link is still not perfect because Docusaurus doesn't accept links to the parent category (you have to give it the URL of the first page... one thing I've never liked about the Git Docs based UI/UX).  Just fixing this in branch now, if I can...

**Review by rdlrt — APPROVED** [review]

**Comment by jinglescode** [comment]
Thank you @rdlrt and @rphair. We need more reviewers. Let me know if you see anything else to fix/add. 
Thanks for your reviews.

**Review by katomm — APPROVED** [review]
👍

**Pull request merged**